AlicerceTests/Extensions/OptionalTests.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  OptionalTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+final class OptionalTests: XCTestCase {
+
+    func testThen_UsingNonOptionalValue_ShouldUnwrapTheValue() {
+        let anOptionalString: String? = "😎"
+
+        anOptionalString.then { value in
+            XCTAssertTrue(anOptionalString! == value, "🚀 \(value) unwrapped ✅")
```

Comment:
🤔 u'r right, we don't. 

1. ok, i'm not used to `XCTest`, for me AssertTrue and AssertEqual are the same, but i understand the differences

2. 👍 

3. 👍